torch/csrc/deploy/interactive_embedded_interpreter.cpp

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
yep. I have one question. I developed this PR under FBCode. It looks like I can not run setup.py under FBCode since the script requires running under a git repository. Do I have to import this PR to a clean git repository to verify the cmake build? Not sure if ghimport is the tool I need to use

Reply:
maybe we can follow up with the cmake build; I don't think it's blocking this PR